.travis.yml

```diff
@@ -1,34 +1,17 @@
-sudo: false
-language: cpp
+sudo: required
 
-matrix:
-  include:
-    - os: linux
-      compiler: clang
-      addons: &clang38
-        apt:
-          sources: ['llvm-toolchain-precise-3.8', 'ubuntu-toolchain-r-test']
-          packages: ['clang-3.8', 'libclang-3.8-dev']
-      env: COMPILER='clang++-3.8'
+language: python
 
-install:
-  - pip install --user clang jinja2 nose
-  - gem install ffi
-  - DEPS_DIR="${TRAVIS_BUILD_DIR}/deps"
-  - mkdir ${DEPS_DIR} && cd ${DEPS_DIR}
-  - CMAKE_URL="http://www.cmake.org/files/v3.3/cmake-3.3.2-Linux-x86_64.tar.gz"
-  - mkdir cmake && travis_retry wget --no-check-certificate --quiet -O - ${CMAKE_URL} | tar --strip-components=1 -xz -C cmake
-  - export PATH=${DEPS_DIR}/cmake/bin:${PATH}
-  - cd ${TRAVIS_BUILD_DIR}
-
-before_script:
-  - export CXX=${COMPILER}
-  - export LD_LIBRARY_PATH=$(dirname $(find /usr/lib/llvm-3.8 -name libclang.so))
+services:
+  - docker
 
-script: ${TRAVIS_BUILD_DIR}/run_tests.sh
-
-notifications:
-  slack:
-    rooms:
-      - c-api:ZTxxqvEcr25iaaB8XBS7luaA#general
+install:
+  - if [ ${TRAVIS_PULL_REQUEST} = "false" ]; then
```

Comment:
Wasn't that the problem in 2c2c7ffaf6fa5a3e1879823851f3c1f004f8dcd3, which is why we reverted dockerised tests in the first place?